src/graph/graph_entity.h

```diff
@@ -25,24 +29,31 @@ typedef struct {
     SIValue value;
 } EntityProperty;
 
+// Essence of a graph entity.
+// TODO: see if pragma pack 0 will cause memory access violation on ARM.
```

Comment:
I think we're incompatible with ARM right now regardless - isn't that why we switched from triemaps to RAX on the previous repo?. Or perhaps that was a 32-bit issue.